### PR TITLE
fix: switch selected component event name list still show the origin one

### LINKED ISSUE #747: 🐛 [Bug]: 画布切换组件后绑定事件列表依然还是原组件的绑定事件列表
### Environment

chrome

### Version

latest

### Version

latest

### Link to minimal reproduction

none

### Step to reproduce

重现步骤：
1.  点击box组件，查看绑定事件列表
2.  点击table组件，查看绑定事件列表
3.  切换回box组件，查看绑定事件列表，事件列表依然时table组件的事件列表
IMG_2400
IMG_2401
IMG_2402

 

### What is expected

_No response_

### What is actually happening

_No response_

### What is your project name

tiny-engine branch：develop

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?
修复画布切换选择组件后，高级面板事件列表仍然显示原组件事件列表的bug

Issue Number: close #747 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Enhanced event management with a new computed property, `renderEventList`, for improved data handling.
	- Streamlined integration of event data, ensuring the event list is always up-to-date with localization support.

- **Bug Fixes**
	- Resolved discrepancies in event data display by consolidating sources into a single reactive property.

- **Refactor**
	- Renamed state variable from `componentEvents` to `customEvents` for clarity and efficiency.
	- Updated event handling logic to improve flexibility and maintainability.

### CODE REVIEW

File: packages/settings/events/src/components/BindEvents.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@
         <div class="action-item bind-action-item">
           <div class="binding-name" @click="openActionDialog(action)">
             <div>
-              {{ action.eventName }}<span>{{ state.componentEvents[action.eventName].label.zh_CN }}</span>
+              {{ action.eventName }}<span>{{ renderEventList[action.eventName]?.label?.zh_CN }}</span>
```
Comment (hexqi) [inline packages/settings/events/src/components/BindEvents.vue]: zh_CN建议改用locale，`label?.[locale]`

File: packages/settings/events/src/components/BindEvents.vue [hunk 2]
```diff
@@ -126,18 +126,18 @@ export default {
 
     const isBlock = computed(() => Boolean(pageState.isBlock))
     const isEmpty = computed(() => Object.keys(state.bindActions).length === 0)
+    const renderEventList = computed(() => ({ ...state.componentEvent, ...state.componentEvents }))
```
Comment (hexqi) [inline packages/settings/events/src/components/BindEvents.vue]: 这两个变量能否换一个名字，难区分容易弄错，并且state.componentEvent看起来也并不是单个事件的意思

File: packages/settings/events/src/components/BindEvents.vue (hunk 1 above)
Comment (chilingling) [reply packages/settings/events/src/components/BindEvents.vue]: Done

File: packages/settings/events/src/components/BindEvents.vue (hunk 2 above)
Comment (chilingling) [reply packages/settings/events/src/components/BindEvents.vue]: Done

**Review by hexqi — APPROVED** [review]

**Pull request merged**